### PR TITLE
Kotlin `2.0.0`. Wrappers `pre.752`. Compilation target `es2015`

### PR DESCRIPTION
**Pull request by porotkin**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,10 +1,9 @@
 author=[Alexander Popoff](https://github.com/aerialist7)
 group=team.karakum
 version=1.0.0
-kotlin.version=1.9.24
-kotlin.wrappers.version=1.0.0-pre.742
+kotlin.version=2.0.0
+kotlin.wrappers.version=1.0.0-pre.752
 kotlin.code.style=official
 kotlin.js.yarn=false
 kotlin.js.compiler=ir
```
Comment (turansky) [inline gradle.properties:8]: Redundant?

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,6 @@
 # Gradle
 .gradle
 build/
+
+# Kotlin 2
+.kotlin
```
Comment (turansky) [inline .gitignore:10]: After `.gradle` please
Comment isn't required

**Review by turansky — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (porotkin) [reply gradle.properties]: yes

File: .gitignore (hunk 2 above)
Comment (porotkin) [reply .gitignore:7]: gotcha

File: build.gradle.kts [hunk 3]
```diff
@@ -34,12 +34,9 @@ kotlin {
...
 }
 
-// TODO: Migrate to `compilerOptions { target.set("es2015") }` after Kotlin `2.0.0`
-//  Details: https://kotlinlang.org/docs/whatsnew-eap.html#new-compilation-target
-// From https://kotlinlang.org/docs/whatsnew19.html#experimental-support-for-es2015-classes-and-modules
 tasks.withType<KotlinJsCompile>().configureEach {
-    kotlinOptions {
-        useEsClasses = true
+    compilerOptions {
+        target.set("es2015")
```
Comment (turansky) [inline build.gradle.kts:39]: `useEsModules()` isn't required?

**Review by turansky — CHANGES_REQUESTED** [review]

**Pull request merged**